Add chain and diamond filters

LosD opened the pull request:
This will describe the difference between chained and "diamonded" filters. Also updates copyright to 2015, and changes a bit of styling.

 Fixes TEAMDK-212

Commit:
Add chain and diamond filters

This will describe the difference between chained and "diamonded" filters. Also updates copyright to 2015, and changes a bit of styling.

 Fixes TEAMDK-212

kaspersorensen commented on src/docbkx/chapter12-getting-started-desktop.xml:
```diff
@@ -270,23 +270,44 @@
-				instructing what needs to be done in current moment to build a valid
-				job.
+				Using "Link to...", it is also possible to wire several filters to a component
+				in a kind of diamond shape. In that case, if any of the the filters are valid,
```
"valid" maybe not so precise.

What about "... if any of the filter requirements are met,"

kaspersorensen replied:
actually: "any" should then be emphasized (italic) IMO

kaspersorensen commented:
LGTM, see the one remark.

Commit:
Add chain and diamond filters

This will describe the difference between chained and "diamonded" filters. Also updates copyright to 2015, and changes a bit of styling.

 Fixes TEAMDK-212

Commit:
Add chain and diamond filters

This will describe the difference between chained and "diamonded" filters. Also updates copyright to 2015, and changes a bit of styling.

 Fixes TEAMDK-212
